wdl2cwl/functions.py

```diff
@@ -0,0 +1,94 @@
+"""WDL Stdlib Functions"""
+
+import re
+from typing import List, Any
+
+import WDL
+from WDL.Expr import Base
+from .errors import WDLSourceLine, ConversionException
+from .expr import (
+    get_expr,
+    get_expr_name,
+    get_expr_apply,
+    get_expr_name_with_is_file_check,
+)
+from .util import ConversionContext, get_input
+
+
+def _add(arguments: List[Base], ctx: ConversionContext, **kwargs: Any) -> str:
```

Comment:
Signature for the functions is:

`fn ( arguments, ctx, **kwargs )` where

- `fn` must match a function from WDL's [stdlib](https://github.com/chanzuckerberg/miniwdl/blob/5d5b7f8e0147278a7a211f13ad863f6586f902b8/WDL/StdLib.py#L306).
- `arguments` is same as before, the list of args for this function
- `**kwargs` whatever else is passed as a dict/param, like that optional flag